.github/workflows/cla.yaml

```diff
@@ -6,31 +6,24 @@ on:
   pull_request_target:
     types: [opened, closed, synchronize]
 
+permissions:
+  actions: write
+  contents: write
+  pull-requests: write
+  statuses: write
+
 jobs:
   cla:
     runs-on: ubuntu-latest
     steps:
-      - name: "Get Team Members"
-        id: team
-        uses: actions/github-script@v6
-        with:
-          github-token: ${{ secrets.ORG_TOKEN }}
-          result-encoding: string
-          script: |
-            const members = await github.paginate(
-              github.rest.orgs.listMembers,
-              { org: "cowprotocol" },
-            );
-            return members.map(m => m.login).join(",");
-
       - name: "CLA Assistant"
-        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
-        uses: contributor-assistant/github-action@v2.1.3-beta
+        if: ((github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target') && github.repository_owner == 'cowprotocol' && github.repository != 'cowprotocol/cla'
```

Comment:
The changes at the end of the line aren't present in the [CLA repo](https://github.com/cowprotocol/cla/) readme. I think they should be consistent with each other (which could mean as well that we update the readme there, but at least `github.repository != 'cowprotocol/cla'` doesn't seem necessary.

Reply:
This is fair, I'll amend the PRs